fred/cogs/levelling.py

```diff
@@ -4,36 +4,43 @@
 import math
 from datetime import *
 
-import nextcord.ext.commands as commands
-from nextcord import DMChannel
+from nextcord import DMChannel, Message, Guild
+from nextcord.ext.commands import MemberNotFound
 
-import config
-from libraries import common
+from .. import config
+from ..libraries import common
+
+logger = common.new_logger(__name__)
 
 
 class UserProfile:
-    def __init__(self, user_id, guild, bot):
+    def __init__(self, user_id: int, guild: Guild):
         self.guild = guild
-        self.bot = bot
         self.user_id = user_id
-        self.logger = logging.Logger("USERPROFILE")
 
-        assert bot.intents.members, "The bot has no member intents, so it cannot do levelling!!"
```

Comment:
Why remove this?

Reply:
The bot will explode on startup when it tries to declare it has intents.members in code, because discord will say no and nextcord will say oh fuck. The assert is redundant.